.github/workflows/continuous-build.yml

```diff
@@ -18,28 +18,32 @@ jobs:
           - macos-latest
           - ubuntu-latest
           - windows-latest
-        java:
-          # Test all OS's with Java 11
-          - 11
         include:
           - os: ubuntu-latest
-            java: 11
+            testAdditionalJavaVersions: true
             coverage: true
-          - os: ubuntu-latest
-            java: 8
     steps:
       - uses: actions/checkout@v2
         with:
           submodules: true
-      - uses: actions/setup-java@v1
+      - id: setup-java-8
```

Comment:
If gradle is supposed to downloaded needed toolchain, why do we provide it manually?

Reply:
Java is required to launch Gradle itself.

Reply:
For that we have java 11 next to this 8

Reply:
Yeah but setup-java doesn't download Java it just extracts from a local VM disk very quickly. Figure it's worth saving some downloading.